src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -9,8 +9,10 @@ class Lotto(val lottoNumbers: Set<LottoNumber>) {
     }
 
     fun match(winningNumber: WinningLotto): LottoPrize {
+        var isBonusMatch = false
         val count = lottoNumbers.intersect(winningNumber.winningLotto).size
-        return LottoPrize.from(count)
+        if (winningNumber.bonusNumber in lottoNumbers) isBonusMatch = true
+        return LottoPrize.from(count, isBonusMatch)
```

Comment:
isBonusMatch 지역 변수 없이 아래와 같이 사용할 수는 없을까요?

```suggestion
        val count = lottoNumbers.intersect(winningNumber.winningLotto).size
        return LottoPrize.from(count, winningNumber.bonusNumber in lottoNumbers)
```

위와 같은 경우 `winningNumber.bonusNumber in lottoNumbers`가 무엇을 의미하는지 알기 어렵기 때문에 별도로 메소드를 분리하거나 할 수 있겠네요.